src/hdprivatekey.js

```diff
@@ -547,14 +547,6 @@ HDPrivateKey.prototype.toString = function() {
   return this.xprivkey;
 };
 
-/**
- * Returns the console representation of this extended private key.
- * @return string
- */
-HDPrivateKey.prototype.inspect = function() {
```

Comment:
Just curious what was the reason for this removal? Sure, console.log() in production code isn't the best thing ever, but this could stay I suppose?

Reply:
I was getting a warning that inspect is deprecated
https://nodejs.org/api/deprecations.html#deprecations_dep0079_custom_inspection_function_on_objects_via_inspect

There would have been a more consistent way to fix but I was preparing for a demo and wanted a quick fix. We should think about what to do with the inspect functions that are in almost all classes

Reply:
Created followup: #11